minio/api.py

```diff
@@ -68,59 +61,60 @@
+    :param endpoint: A string of the URL of the cloud storage server.
+    :param access_key: Access key to sign self._http.request with.
+    :param secret_key: Secret key to sign self._http.request with.
+    :return: :class:`Minio <Minio>` object
+    """
     def __init__(self, endpoint, access_key=None, secret_key=None):
-        """Constructs a :class:`Minio <Minio>`.
-
-        Examples:
-          client = Minio('https://play.minio.io:9000')
-          client = Minio('https://s3.amazonaws.com',
-                         'ACCESS_KEY', 'SECRET_KEY')
-
-        :param endpoint: A string of the URL of the cloud storage server.
-        :param access_key: Access key to sign self._http.request with.
-        :param secret_key: Secret key to sign self._http.request with.
-        :return: :class:`Minio <Minio>` object
-        """
         is_valid_endpoint(endpoint)
-
         url_components = urlsplit(endpoint)
         self._region_map = dict()
         self._endpoint_url = url_components.geturl()
         self._access_key = access_key
         self._secret_key = secret_key
-        self._user_agent = 'minio-py/{0} ({1}; {2})'.format(__version__,
-                                                            platform.system(),
-                                                            platform.machine())
-
+        self._user_agent = _LIBRARY_AGENT
         self._http = urllib3.PoolManager(
             cert_reqs='CERT_REQUIRED',
             ca_certs=certifi.where()
         )
 
-    # Client level
-    def set_app_info(self, name, version, comments=None):
+    # Set application information.
+    def set_app_info(self, app_name, app_version):
         """
-        Adds an entry to the list of user agents.
+        Sets your application name and version to
+        default user agent in the following format.
+
+              Minio (OS; ARCH) LIB/VER APP/VER
 
         Example:
-            client.set_app_info('my_app', '1.0.0', ['ex', 'parrot'])
-            # Results in my_app/1.0.0 (ex; parrot) appended to user agent
+            client.set_app_info('my_app', '1.0.0')
 
-        :param name: application name.
-        :param version: application version.
-        :param comments: list of comments to include in comments section.
+        :param app_name: application name.
+        :param app_version: application version.
         """
-        if name == '' or version == '':
+        if app_name == '' or app_version == '':
             raise ValueError
 
-        if comments is not None:
-            joined_comments = '; '.join(comments)
-            components = [' ', name, '/', version, ' (', joined_comments, ')']
-            self._user_agent += ''.join(components)
-        else:
-            components = [' ', name, '/', version, ' ']
-            self._user_agent += ''.join(components)
+        app_info = ' {0}/{1}'.format(app_name,
+                                     app_version)
+        self._user_agent = _LIBRARY_AGENT + app_info
```

Comment:
it would be better like
```python
app_info = "{0}/{1}".format(app_name, app_version)
self._user_agent = " ".join(_LIBRARY_AGENT, app_info)
```
if we support multiple app_info in future this will help